tensorflow/core/grappler/optimizers/remapper_test.cc

```diff
@@ -449,6 +449,76 @@ TEST_F(RemapperTest, FuseMatMulWithBias) {
   test::ExpectTensorNear<float>(tensors[0], tensors_expected[0], 1e-6);
 }
 
+TEST_F(RemapperTest, FuseConv2DWithBiasAndActivationOnGPU) {
+  using ::tensorflow::ops::Placeholder;
```

Comment:
This test has to be skipped if there are no GPUs available.

Reply:
> This test has to be skipped if there are no GPUs available.

Done, thanks!